var/spack/repos/builtin/packages/pngquant/package.py

```diff
@@ -16,3 +16,5 @@ class Pngquant(AutotoolsPackage):
     url      = "http://pngquant.org/pngquant-2.12.5-src.tar.gz"
 
     version('2.12.5', sha256='3638936cf6270eeeaabcee42e10768d78e4dc07cac9310307835c1f58b140808')
+
+    depends_on('libpng', type='build')
```

Comment:
It's very strange to have a library as a `build` only dependency. Are you sure this is correct?

Reply:
I just tested install/uninstall/help function, if there is no `libpng`, `configure` scripts will report error.
but with `type='build'`, it can bootstrap well, I did not test the further function.
I will change the `libpng` dpendency to `build` and `run`, just in case some function will use it.